snow/engine/snowman/bootstrap/bootstrapper_test.go

```diff
@@ -1510,3 +1502,127 @@ func TestBootstrapContinueAfterHalt(t *testing.T) {
 		t.Fatal("Should have left blk1 as missing")
 	}
 }
+
+func TestBootstrapNoParseOnNew(t *testing.T) {
+	require := require.New(t)
+
+	ctx := snow.DefaultConsensusContextTest()
+
+	peers := validators.NewSet()
+
+	sender := &common.SenderTest{}
+	vm := &block.TestVM{}
+
+	sender.T = t
+	vm.T = t
+
+	sender.Default(true)
+	vm.Default(true)
+
+	isBootstrapped := false
+	bootstrapTracker := &common.BootstrapTrackerTest{
+		T: t,
+		IsBootstrappedF: func() bool {
+			return isBootstrapped
+		},
+		BootstrappedF: func(ids.ID) {
+			isBootstrapped = true
+		},
+	}
+
+	sender.CantSendGetAcceptedFrontier = false
+
+	peer := ids.GenerateTestNodeID()
+	if err := peers.Add(peer, nil, ids.Empty, 1); err != nil {
+		t.Fatal(err)
```

Comment:
any reason to not use `require.NoError` here?

Reply:
CTRL+C -> CTRL+V was the reason

Reply:
addressed